src/test/java/teammates/test/cases/browsertests/StudentCourseJoinConfirmationPageUiTest.java

```diff
@@ -14,9 +14,8 @@
 import teammates.test.pageobjects.StudentHomePage;
 
 /**
- * SUT: {@link Const.ActionURIs#STUDENT_COURSE_JOIN},
- *      {@link Const.ActionURIs#STUDENT_COURSE_JOIN_AUTHENTICATED},
- *      {@link Const.ActionURIs#STUDENT_COURSE_JOIN_NEW}.
+ * SUT: {@link Const.WebPageURIs#JOIN_PAGE},
+ *      {@link Const.WebPageURIs#STUDENT_HOME_PAGE}.
```

Comment:
I don't think `Const.WebPageURIs#STUDENT_HOME_PAGE` is relevant here.

Reply:
Removed it from the Javadoc.